README.md

```diff
@@ -114,10 +114,10 @@ client, _ := whisk.NewClient(http.DefaultClient, nil)
 
 options := &whisk.ActionListOptions{
   Limit: 30,
-  Skip: 30,
+  Skip: 0,
```

Comment:
Skippin g 30 would be silly for someone "getting started"; good change.

Reply:
I would change the limit to 10 as this seems like what would be a logical default for many downstream tools that use the client...